llama_stack/templates/nvidia/run.yaml

```diff
@@ -135,6 +135,13 @@ models:
   provider_id: nvidia
   provider_model_id: meta/llama-3.2-90b-vision-instruct
   model_type: llm
+- metadata: {
+    embedding_dimension: 1024
+  }
```

Comment:
nit: these run.yaml templates are generated via running 

```
python llama_stack/scripts/distro_codegen.py 
```

we could modify the file here https://github.com/meta-llama/llama-stack/blob/7558678b8cfc012766d70b38e5d9890937d14598/llama_stack/templates/nvidia/nvidia.py#L19 and run script to generate the run.yaml file.